js/tji-datasets-explore.js

```diff
@@ -494,28 +506,36 @@ TJIChartView.prototype.create_filter_checkbox = function(name, value, id) {
     });
   var label = jQuery('<label/>', {
     for: 'TJIChartView__filter-' + name + '-' + value,
-  }).text(value);
+  }).text(value.toLowerCase());
   var container = jQuery('<div/>', {
       id: id,
-      class: "tji-chart-filters__filter",
+      class: "tji-chartview-filters__filter",
   }).append(input, label);
   return container;
 }
 
 TJIChartView.prototype.create_filter_autocomplete = function(filter) {
   var that = this;
-  var fieldset = jQuery('<fieldset><legend>' + filter.name.replace(/_/g, " ") + '<i class="fas fa-caret-down"></i></legend></fieldset>');
+
+  var fieldset = jQuery('<fieldset/>');
+  var legend = this.create_filter_legend(filter.name);
+  var filterset = jQuery('<div/>', {
```

Comment:
filterset wraps the portion of the filters that needs to be collapsed or expanded.